core/planner.cpp

```diff
@@ -45,23 +47,19 @@ static int decostoplevels_imperial[] = { 0, 3048, 6096, 9144, 12192, 15240, 1828
 #if DEBUG_PLAN
 void dump_plan(struct diveplan *diveplan)
```

Comment:
I suggest changing the argument to `const struct diveplan &diveplan)` because
1) I don't see why it would make sense to allow a NULL ptr.
2) The `dump()` function hopefully doesn't modify the plan, does it?